src/embed.js

```diff
@@ -72,7 +72,7 @@ function toHTML({
   // there is a provider preference, let's go with it.
   if (oEmbed.html) {
     const html = decorateHTML(oEmbed.html, title);
-    return `<div class="embed embed-oembed ${kind}" data-url="${fallbackURL}">
+    return `<div class="embed embed-oembed ${kind}" data-url="${encodeURI(fallbackURL)}">
```

Comment:
no, this is wrong. it needs to be a HTML iso escape (the one with the `&amp;`)

Reply:
Of course...